tests/unit/cloud/clouds/test_digitalocean.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""
+    :codeauthor: `Gareth J. Greenaway <gareth@saltstack.com>`
+
+    tests.unit.cloud.clouds.digitalocean_test
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+
+# Import Salt Libs
+from salt.cloud.clouds import digitalocean
+from salt.exceptions import SaltCloudSystemExit
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock
+from tests.support.unit import TestCase
+
+log = logging.getLogger(__name__)
+
+
+class DigitalOceanTestCase(TestCase, LoaderModuleMockMixin):
+    """
+    Unit TestCase for salt.cloud.clouds.digitalocean module.
+    """
+
+    def setup_loader_modules(self):
+        return {digitalocean: {"__virtual__": MagicMock(return_value="digitalocean")}}
```

Comment:
Looks like something has changed to prevent this:

```
E                       RuntimeError: No need to patch '__virtual__'. Passed loader module dict: {<module 'salt.cloud.clouds.digitalocean' from '/home/ken/working/git/github/saltstack/salt-five/salt/cloud/clouds/digitalocean.py'>: {'__virtual__': <MagicMock id='140673177908896'>}}
```

Will remove.